src/Styles/app.css

```diff
@@ -0,0 +1,50 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.nav-bar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-start;
+  padding: 1rem;
+  margin-bottom: 15rem;
+}
+
+.links {
+  margin-top: 1.9rem;
+  text-decoration: none;
+  font-size: 1rem;
+  font-weight: bold;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.para1 {
+  font-size: 1rem;
+  font-weight: 500;
+  gap: 2rem;
+  margin-left: 10px;
+}
```

Comment:
- Let's also move the paragraph to the middle of the page to be close to the wireframe.

![Screenshot from 2023-01-19 22-02-04](https://user-images.githubusercontent.com/59397159/213560246-e4295675-1da6-4d9b-9040-5e2b9934401a.png)